python/graphstorm/gpartition/dist_partition_graph.py

```diff
@@ -189,12 +190,33 @@ def main():
             dirs_exist_ok=True,
         )
 
+    # Hard Negative Mapping
+    if args.gsprocessing_config:
+        gsprocessing_config = args.gsprocessing_config
+        shuffle_hard_negative_nids(f"{args.input_path}/{gsprocessing_config}",
```

Comment:
How do you know, there is a hard negative feature?

Reply:
``shuffle_hard_negative_nids`` will first check if there is a hard negative transformation, if no matched, skip.